content/en/logs/log_collection/javascript.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Browser Log Collection
+kind: documentation
+aliases:
+  - /logs/log_collection/web_browser
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+- link: "logs/processing/parsing"
+  tag: "Documentation"
+  text: "Learn more about parsing"
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "logs/explorer/analytics"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "logs/faq/log-collection-troubleshooting-guide"
+  tag: "FAQ"
+  text: "Log Collection Troubleshooting Guide"
+---
+
+
+
+Send log to Datadog from web browsers or other Javascript client thanks to the Datadog Client-side JavaScript logging library.
+
+With the `browser-agent-core` library you are able to send log directory to Datadog from JS clients and leverage the following features:
```

Comment:
`browser-agent-core` is maybe not relevant anymore since we change the file name to `datadog-logs`

Reply:
Good point, forgot to change the name of the library when I updated the link. Will do